infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerRuntimeInfrastructure.java

```diff
@@ -80,7 +80,7 @@ protected RuntimeContext internalPrepare(
 
   @Override
   public Set<RuntimeIdentity> getIdentities() throws InfrastructureException {
-    return containers.findIdentities();
```

Comment:
I think it would be better to comment old line and add a link to issue why docker recovery is disabled https://github.com/eclipse/che/issues/5814.

Reply:
Ok